### PR TITLE
Add `PagerankTable` for exploring PageRank results

### PR DESCRIPTION
**Pull request by decentralion**
`PagerankTable` is forked from `ContributionList`.

Test plan: I took it for a spin and it seemed OK. I'm not inclined to
write formal tests because we are in rapid prototyping mode still.

### CODE REVIEW

File: src/app/credExplorer/App.js [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+  pagerankResult: ?PagerankResult,
 };
 
 export default class App extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      repoOwner: "",
-      repoName: "",
+      repoOwner: "sourcecred",
```
Comment (wchargin) [inline src/app/credExplorer/App.js]: remove

File: src/app/credExplorer/App.js [hunk 2]
```diff
@@ -33,28 +35,36 @@ export default class App extends React.Component<Props, State> {
           <h1>Cred Explorer</h1>
         </header>
         <p>Welcome to the SourceCred Explorer!</p>
+        <div id="PageRankContainer">
```
Comment (wchargin) [inline src/app/credExplorer/App.js]: Let's remove this spurious `div` and its brethren below.

File: src/app/credExplorer/pagerankTable.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    const nodesByScore = graph
+      .nodes()
+      .slice()
+      .sort((a, b) => {
+        const x = pagerankResult.get(a.address).probability;
+        const y = pagerankResult.get(b.address).probability;
+        return x - y;
+      })
+      .reverse()
+      .filter(
```
Comment (wchargin) [inline src/app/credExplorer/pagerankTable.js]: For our small data set, it doesn't matter too much, but I think that it makes more sense to `filter` before you `sort`. You wouldn't want to sort billions of nodes if you're only going to display a handful of 'em.

File: src/app/credExplorer/pagerankTable.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      </label>
+    );
+  }
+
+  renderTable() {
+    if (this.props.graph == null || this.props.pagerankResult == null) {
+      throw new Error("Impossible.");
+    }
+    const graph = this.props.graph;
+    const pagerankResult = this.props.pagerankResult;
```
Comment (wchargin) [inline src/app/credExplorer/pagerankTable.js]: Usually people will write `const {graph, pagerankResult} = this.props;` here.

Comment (decentralion) [reply src/app/credExplorer/pagerankTable.js]: Done

**Pull request merged**